### PR TITLE
RH and OTA wizard enhancements

### PR DESCRIPTION
**Pull request by szafranski**
defaults to default value when just enter is pressed, code is nicer now

### CODE REVIEW

**Comment by szafranski** [comment]
If U agree with those changes please merge and delete a branch.

**Comment by szafranski** [comment]
you can check it now. some cleaning preasumebly can be done but should be ok

**Comment by szafranski** [comment]
I cannot stop myself :) But now it should be "final"

File: conf_wizard_ota.py [hunk 1]
```diff
@@ -54,33 +54,44 @@ def do_config(config):
...
+        else:
+            config.pi_user = pi_user_name
         while True:
             version = input(f"\nChoose RotorHazard version? \
 [{Bcolors.UNDERLINE}stable{Bcolors.ENDC} | beta | master ]\t\t\t").lower()
-            version_valid_options = ['master', 'stable', 'beta', 'custom']
-            if version not in version_valid_options:
-                print("\nPlease enter correct value!")
-            else:
+            if len(version) == 0:
```
Comment (DavidJJ) [inline conf_wizard_ota.py:68]: this can just be
if not version:  
instead of
if len(version) == 0

File: conf_wizard_ota.py [hunk 2]
```diff
@@ -54,33 +54,44 @@ def do_config(config):
...
         while True:
             country_code = input("\nWhat is your country code? [default: GB]\t\t\t\t").upper()
-            if 1 < len(country_code) <= 3:
-                config.country = country_code
-                break
-            else:
+            if len(country_code) > 4:
                 print("\nPlease enter correct value!")
+                continue
+            elif len(country_code) == 0:
```
Comment (DavidJJ) [inline conf_wizard_ota.py:89]: same as above

Comment (szafranski) [reply conf_wizard_ota.py:89]: like... Really? :)

**Comment by szafranski** [comment]
Will change those than. But general "asspumptions" and "mechanics" are ok?

**Pull request merged**